app/editor/src/features/admin/notifications/dashboard/utils/getLastSent.ts

```diff
@@ -0,0 +1,9 @@
+import { INotificationInstanceModel } from 'tno-core';
+
+import { formatDate } from './formatDate';
+
+export const getLastSent = (instance: INotificationInstanceModel) => {
```

Comment:
Seeing duplicating with subscriber and editor code.  Ideally we'd move this to `tno-core` and update each app to use the shared identical method.

Reply:
This function is different than the report one. This one receive a INotificationInstanceModel, the report receives a IReportModel.